app/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
     compileSdkVersion 33
     namespace 'com.poupa.vinylmusicplayer'
     defaultConfig {
-        minSdkVersion 16
+        minSdkVersion 19
```

Comment:
This is important: The min API version is now 19, due to MaterialCab requirement.

Reply:
If I understand [this](https://apilevels.com/) properly, we're dropping support for Android 4.1/4.2/4.3 and require 4.4 from now on. This is perfectly acceptable imo